src/index.ts

```diff
@@ -6,4 +6,5 @@ export { isUndefined } from './isUndefined';
 export { loadScript } from './loadScript';
 export { storage } from './storage';
 export { timeAgo } from './timeAgo';
+export { getCookie } from './cookies';
```

Comment:
i think this is missing some exports?

Reply:
Exported all of them now! Ready to be used as a replacement module of [frontend cookies.js](https://github.com/guardian/frontend/blob/main/static/src/javascripts/lib/cookies.js)